seekpath/getpaths.py

```diff
@@ -325,3 +325,268 @@ def get_explicit_k_path(
+
+        - ``cell`` is a :math:`3 \times 3` list of floats (``cell[0]`` is the first lattice
+          vector, ...)
+        - ``positions`` is a :math:`N \times 3` list of floats with the atomic coordinates
+          in scaled coordinates (i.e., w.r.t. the cell vectors)
+        - ``numbers`` is a length-:math:`N` list with integers identifying uniquely
+          the atoms in the cell (e.g., the Z number of the atom, but
+          any other positive non-zero integer will work - e.g. if you
+          want to distinguish two Carbon atoms, you can set one number
+          to 6 and the other to 1006)
+
+    :param with_time_reversal: if False, and the group has no inversion
+        symmetry, additional lines are returned as described in the HPKOT
+        paper.
+
+    :param recipe: choose the reference publication that defines the special
+       points and paths.
+       Currently, the following value is implemented:
+
+       - ``hpkot``: HPKOT paper:
+         Y. Hinuma, G. Pizzi, Y. Kumagai, F. Oba, I. Tanaka, Band structure
+         diagram paths based on crystallography, Comp. Mat. Sci. 128, 140 (2017).
+         DOI: 10.1016/j.commatsci.2016.10.015
+
+    :param threshold: the threshold to use to verify if we are in
+        and edge case (e.g., a tetragonal cell, but ``a==c``). For instance,
+        in the tI lattice, if ``abs(a-c) < threshold``, a
+        :py:exc:`~seekpath.hpkot.EdgeCaseWarning` is issued.
+        Note that depending on the bravais lattice, the meaning of the
+        threshold is different (angle, length, ...)
+
+    :param symprec: the symmetry precision used internally by SPGLIB
+
+    :param angle_tolerance: the angle_tolerance used internally by SPGLIB
+
+
+    :return: a dictionary with the following
+      keys:
+
+        - ``point_coords``: a dictionary with label -> float coordinates
+        - ``path``: a list of length-2 tuples, with the labels of the starting
+          and ending point of each label section
+          input crystal structure has inversion symmetry or not.
+        - ``augmented_path``: if True, it means that the path was
+          augmented with the :math:`-k` points (this happens if both
+          has_inversion_symmetry is False, and the user set
+          with_time_reversal=False in the input)
+        - ``is_supercell``: True if the input unit cell is a supercell of
+          a smaller primitive cell.
+
+    :note: An :py:exc:`~seekpath.hpkot.EdgeCaseWarning` is issued for
+        edge cases (e.g. if ``a==b==c`` for
+        orthorhombic systems). In this case, still one of the valid cases
+        is picked.
+    """
+
+    res = get_path(
+        structure=structure,
+        with_time_reversal=with_time_reversal,
+        threshold=threshold,
+        symprec=symprec,
+        angle_tolerance=angle_tolerance,
+        recipe=recipe,
+    )
+
+    is_supercell = abs(res["volume_original_wrt_prim"] - 1) > 0.1
+
+    if is_supercell:
+        print(
```

Comment:
Replace with a warning instead, so it can be easily filtered

Reply:
Done